SystemServices/SystemServices.h

```diff
@@ -134,8 +134,11 @@ namespace WPEFramework {
                 void InitializeIARM();
                 void DeinitializeIARM();
 #endif /* defined(USE_IARMBUS) || defined(USE_IARM_BUS) */
-
+                
+                
                 /* Events : Begin */
+               
+                static int FwUpdateState_LatestEvent;
```

Comment:
This doesn't need to be static. There can only be on instance of a plugin running in thunder. 
Pls change it to non-static private data member. Also pls change the other static members to be non-static.

static std::string m_currentMode;
static cTimer m_operatingModeTimer;
static int m_remainingDuration;